index.html

```diff
@@ -12,8 +12,107 @@
+    Information Technology/Programming/Telecommunications</h3>
+
+<div class = "line">
 
-<body>
-  <h1>My Resume</h1>
+</section> 
+
+<div  class = "one">
+<h6>Education</h6>
+
+   <ul>
+     <li>AMERICAN INTERCONTINENTAL UNIVERSITY,
+  Hoffman Estates, IL 10/05 – 11/06
+  Bachelor of Science in Information Technology
+  Major: Information Technology/Computer Programming
+  Magna cum Laude/GPA of 3.7 // 90 credits earned </li>
+  <br>
+  <li>VOORHEES COLLEGE,
+  Denmark, S.C. 08/02 – 05/05
+  Major: Computer Science // Minor: Mathematics-- Held a GPA
+  of 3.5, Honors College. transferred
+  89 credits earned</li>
+  <br>
+  <li>Western Governor’s University
+  Bachelors of Science in Computer Science
+  Anticipated Graduation Date: 10/22</li>
+  <br>
+  <li>Lambda School
+  Full-Stack Web Development & Computer Science
+  Anticipated Graduation Date: 06/20</li>
+  </ul>
+
+<br>
+
+
+<h3>Skills</h3>
+
+<div class = "two">
+<p>Works effectively as an individual or with a team, strong communication & organizational skills, exceptionally detail oriented, skilled in
+  Information Technology/Programming/Telecommunications 
+  </p>
+<br>
+
+<div class = "three">
+
+    <h4>Work History</h4>
+ 
+      <ul>
+          <li>Windstream,
+              LNP Specialist I / LNP Specialist II / Provisioning Specialist II <br>
+                Process inbound local number portability requests. Complete internal system research to confirm local service requests. Make calls for
+              status/escalations of LSR's. Technical writing of training documents, training, order entry, provisioning responsibilities. Systems: Metasolv/Oracle,
+              ICARE, WinCARE, Neustar, Aptis, BPMS, CAMS, DCRIS, JIRA, Salesforce.
+               </li>
+       <br>
+       <li>CommunityConnect
+          Sr. Project Coordinator <br>
+          Administrative support for telecommunications start-up project/company. Coordinate customer orders, website design (Frontpage, Macromedia,
+          Photoshop, HTML, CSS, and JavaScript), Cisco equipment configuration/provisioning, local number portability, help desk support/ticketing,
+          technical writing of customer facing manuals. Systems: CRM, Office Applications 
+          </li>
+       <br>
+       <li>US Department of Agriculture <br>
+        Software Engineer - Intern
+        Produced software program for color sorting peanut hulls. Systems/Programs: Perl, VBA, SQL, C++ and Way 2-C</li>
+       <br>
+       <li>Lion Bridge <br>
+          Social Media Evaluator - remote
+          Quality Control inspection tasks for Facebook Ads. Test functionality and relevancy of Facebook ads </li>
+       </ul>
+      
+       <footer>
+          <div class = "footer-content">
+          Copyright Carmen Salley Resume! 2019 <br><br>
+          Contact: csalley@gmail.com (864) 777-9311
+        </footer>
```

Comment:
You could use [tel](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#Creating_a_phone_link) and [mailto](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#Creating_an_email_link) links here to let users easily open an email or text message to you.